README.md

```diff
@@ -341,7 +341,7 @@ A connection object can be used later as an argument to the `erase` member
 function of the resource to remove the listener.<br/>
 There exists also the `clear` member function to drop all the listeners at once.
 
-Almost all the resources use to emit `ErrorEvent` events in case of errors.<br/>
+Almost all the resources use `ErrorEvent` to emit events in case of errors.<br/>
```

Comment:
`ErrorEvent` isn't a function, it's the emitted event(s). I think the meaning changed here.

Reply:
got it...will push an update.